src/modules/pool.module.ts

```diff
@@ -2,6 +2,10 @@ import dotenv from 'dotenv';
 dotenv.config();
 
 import pkg from 'pg';
+import config from '../config';
```

Comment:
Did this work for you in local? 

The way I am currently set-up with my transpiler, I need this to be `../config.js` to run.